recipes/sqlalchemy-redshift/meta.yaml

```diff
@@ -0,0 +1,44 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+{% set name = "sqlalchemy-redshift" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/sqlalchemy-redshift/sqlalchemy-redshift.git
+  git_rev: "{{ version }}"
```

Comment:
Please do not use git for the package as it uses up extra CI time and resources.  Instead, grab the tarball from github. Thanks!